### PR TITLE
Issue-Fix-Critical | Fix nginx-es image and Check CRITICAL vulnerability

### PR DESCRIPTION
**Pull request by santoshr1016**
## Description

Update the elastic-es docker image.
Scan only CRITICAL Vulnerabilities

## Related Issues
https://github.com/astronomer/astronomer/pull/1558

Do not merge this PR until this text is replaced with links to related issues.

## Testing

Do not merge this PR until this text is replaced with details about how these changes were tested.

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Comment by pgvishnuram** [comment]
HIGH, CRITICAL is fine - let the flag be there

File: bin/trivy-scan.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ set +exo pipefail
 trivy \
   --cache-dir /tmp/workspace/trivy-cache \
   image \
-  -s HIGH,CRITICAL \
+  -s CRITICAL \
```
Comment (pgvishnuram) [inline bin/trivy-scan.sh:15]: @santoshr1016 - let it be HIGH,CRITICAL

Comment (danielhoherd) [inline bin/trivy-scan.sh:15]:
```suggestion
  -s HIGH,CRITICAL \
```

**Comment by pgvishnuram** [comment]
@santoshr1016 - resolve conficts on this branch

**Review by pgvishnuram — APPROVED** [review]

**Pull request merged**